src/core/string.c

```diff
@@ -56,6 +56,9 @@ uint8_t *string_copy(const uint8_t *src, uint8_t *dst, int maxlength)
 
 int string_length(const uint8_t *str)
 {
+    if (!str)
+        return 0;
+    
```

Comment:
```suggestion
    if (!str) {
        return 0;
    }
```

We always use the curvy braces even for single-liners.